### PR TITLE
New pudge flesh heaps

### PR DESCRIPTION
**Pull request by Wouterz90**

### CODE REVIEW

**Comment by darklordabc** [comment]
Will test these in a bit.

**Comment by darklordabc** [comment]
@Wouterz90 cooldown reduction is not working at all when I tested it.  The others work great.

File: src/game/scripts/vscripts/abilities/pudge_flesh_heap_cooldown_reduction.lua [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    self:SetStackCount( self:GetFleshHeapKills() )
+    self:GetParent():CalculateStatBonus()
+  end
+end
+
+--------------------------------------------------------------------------------
+
+function modifier_flesh_heap_cooldown_reduction:DeclareFunctions()
+  local funcs = {
+    MODIFIER_PROPERTY_EVASION_CONSTANT,
```
Comment (TheRisen41) [inline src/game/scripts/vscripts/abilities/pudge_flesh_heap_cooldown_reduction.lua]: I think "MODIFIER_PROPERTY_EVASION_CONSTANT" should be something like "MODIFIER_PROPERTY_COOLDOWN_PERCENTAGE"

Comment (Wouterz90) [reply src/game/scripts/vscripts/abilities/pudge_flesh_heap_cooldown_reduction.lua]: Hmm glad it's something simple

**Pull request merged**